common/environment/setup/install.sh

```diff
@@ -258,3 +258,34 @@ _vmkdir() {
 		install -dm${mode} ${_destdir}/${dir}
 	fi
 }
+
+_vcompletion() {
+	local cmd="$1" shell="$2" file="$3"
```

Comment:
If you want default something, `${cmd:=${pkgname}}` is a hit quite often.

Reply:
Sounds good.